ontimize-boot-keycloak/pom.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>ontimize-boot</artifactId>
+        <groupId>com.ontimize.boot</groupId>
+        <version>2.1.3</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>ontimize-boot-keycloak</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
```

Comment:
If ontimize-boot-keycloak is a module of ontimize-boot., the version must be the same as its parent